File: public/server/go/src/usr/testG.go
package usr

import (
	"GoPluginMagicModule/src/globalThis"
	"GoPluginMagicModule/src/helper"

	"github.com/lockval/go2plugin"
)

func (e export) Export_testG(input *go2plugin.Input) map[string]any {

	Sum := helper.Any2Func[func(string, string) string](globalThis.G["lockval"], "Sum")
	c := Sum("a", "b")
	return map[string]any{"c": c, "lang": "Go"}
}

File: public/server/lua/src/usr/testG.lua
return require("umd").define({
    "exports",
}, function(exports)

    function exports.main(input)


        local c = _G.G["lockval"].Sum("a","b")
        local resp={}
        resp["c"]=c
        resp["lang"]="Lua"
        return resp
    end

end)
